os/registry-authentication.md

```diff
@@ -0,0 +1,34 @@
+# Using authentication for a registry
+
+To pull images behind an authentication wall, you need to setup some form of authentication with your image registry like [quay.io][quay-site] or [hub.docker.com][docker-hub-site].
+
+This document outlines the preferred method of authentication with [quay.io][quay-site] and [hub.docker.com][docker-hub-site] and how programmaticaly setup this authentication to your hosts running Docker, Kubernetes, rkt, and Mezos.
+
+## Using a Quay robot for registry auth
+
+The preferred way to setup registry authentication with [quay.io][quay-site] is to use a [Quay Robot][quay-robot]. This essentially acts as a login-token wth some nice features including:
+
+- Pre-populated repository-auth configuration files.
+- Per-repository permissions
+- Token regeneration
+
+![Quay Robot settings][quay-bot-img]
+
+[comment]: # (Yes, this robot has been deleted. Un-obscured to help maintain educational value.)
+
+For more information, check out the [Quay robot documentation][quay-robot].
+
+## Manual registry auth setup
+
+### Docker hub
```

Comment:
Docker Hub is a registry; the others in this list are properly not. They may belong in this document, but not at the identical level as a registry *to which* one authenticates.